dev-python/mamba/mamba-0.11.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
One empty line below EAPI please (not going to ask you to fix other ebuilds for this).

Reply:
Might as well do this for all while I'm at it.